mitmproxy/addons/streambodies.py

```diff
@@ -28,12 +29,18 @@ def run(self, f, is_request):
             if expected_size and not r.raw_content and not (0 <= expected_size <= self.max_size):
                 # r.stream may already be a callable, which we want to preserve.
                 r.stream = r.stream or True
-                # FIXME: make message generic when we add rquest streaming
-                ctx.log.info("Streaming response from %s" % f.request.host)
+                ctx.log.info("Streaming {} {}".format("response from" if not is_request else "request to", f.request.host))
 
-    # FIXME! Request streaming doesn't work at the moment.
     def requestheaders(self, f):
         self.run(f, True)
 
     def responseheaders(self, f):
         self.run(f, False)
+
+    def websocket_start(self, f):
+        if ctx.options.stream_websockets and isinstance(f, websocket.WebSocketFlow):
+            f.stream = True
+            ctx.log.info("Streaming WebSocket message {client} - {server}".format(
```

Comment:
From here on we are streaming all messages in this flow. The info text could be misleading.
Maybe rephrase to something like: **Streaming all WebSocket messages between {client} and {server}**